src/game/scenes/Game.js

```diff
@@ -4,20 +4,61 @@ import AnimatronicsNames from '../utils/AnimatronicsNames';
 
 import Animatronic from '../entities/Animatronic';
 
```

Comment:
You shouldn't leave this lime empty, import Cup here.

Here's how we work with imports:
1. Firstly we import Phaser, cause that's the engine that we work with and it's basically the main thing.
2. Then we import our utilities that help us to work with our game objects.
3. Then we import our game objects themselves. We import them in the order that they appear in the code and we don't leave blank lines between them, cause they're kinda one import group.